pkg/util/util.go

```diff
@@ -111,23 +111,22 @@ func ValidateField(data any, spec rules.ElementRule, mask string) error {
 	dataStr := strings.TrimSpace(fmt.Sprintf("%v", data))
 
 	switch mask {
-	case rules.MASK_REQUIRED:
-		if dataStr == "" {
-			// TODO
-			fmt.Errorf("the element is require")
-		}
-		break
-	case rules.MASK_NOTUSED:
-		if dataStr != "" {
-			// TODO
-			fmt.Errorf("the element is not used")
-		}
-		break
+	/*
+		case rules.MASK_REQUIRED:
+			if dataStr == "" {
+				// TODO
+				return errors.New("the element is require")
+			}
```

Comment:
Why comment this out? Removing `break` is all that's needed.

Reply:
we need consider the code carefully.
to be honest, there isn't any describe for this logic, so I want to fix this code after discuss real use cases with customers

Reply:
Sounds good to me.